app/src/main/java/com/greenaddress/greenbits/ui/MainFragment.java

```diff
@@ -136,14 +139,26 @@ public void onClick(final View v) {
             reloadTransactions(false, true);
         }
 
+        mSwipeRefreshLayout = UI.find(mView, R.id.mainTransactionListSwipe);
+        mSwipeRefreshLayout.setColorSchemeColors(ContextCompat.getColor(getContext(), R.color.accent));
+        mSwipeRefreshLayout.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+                Log.d(TAG, "onRefresh -> " + TAG);
+                updateBalance();
+                reloadTransactions(false, false);
+            }
+        });
+
         registerReceiver();
         return mView;
     }
 
     @Override
     protected void onBalanceUpdated() {
+        Log.d(TAG, "onBalanceUpdated -> " + TAG);
```

Comment:
Its useful for some kinds of grepping and filtering when determining sequences of events, feel free to leave it there.